vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -36,6 +36,7 @@ std::vector<int64_t> GetTagsForBranch(INativeUIManagerHost *host, int64_t tag);
 
 TouchEventHandler::TouchEventHandler(const Mso::React::IReactContext &context, bool fabric)
```

Comment:
We need to modify this to also prevent firing onMouseEnter / onMouseLeave events for views that aren't in the scope of the TouchEventHandler